spec/host_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "./lib/host.rb"
+
+describe Host do
+  subject(:host) {described_class.new(menu, order)}
+  let(:order) {double(:order)}
+  let(:menu) {double(:menu)}
+  let(:item) {double(:item)}
+
+  before do
+    allow(menu).to receive(:show_items).and_return("A1....Salmon roll....£7.0")
+    allow(order).to receive(:add_item).and_return(item)
+    allow(order).to receive(:check_order).and_return("A1....Salmon roll....£7.0")
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)